desktop/cmp/dash/canvas/DashCanvasModel.js

```diff
@@ -162,6 +173,7 @@ export class DashCanvasModel extends HoistModel {
     loadState(state) {
         this.clear();
         state.forEach(state => this.addView(state.viewSpecId, state));
+        this._publishStateImmediately();
```

Comment:
why do we need to round-trip this immediately?  looks suspicious

Reply:
Also agreed-- I'll try to come up with a better approach.  This is in response to this bug:

Initial state is not set on state member for DashCanvasModel until 1 second after model creation due to debounce